mesheryctl/internal/cli/root/system/start.go

```diff
@@ -173,6 +179,22 @@ func start() error {
 			}
 		}
 
+		var endpoint meshkitutils.HostPort
+		endpoint.Address = "http://localhost"
+		tempPort, err := strconv.Atoi(userPort[len(userPort)-1])
+		if err != nil {
+			return err
+		}
+		endpoint.Port = int32(tempPort)
+
+		var mockEndpoint *meshkitutils.MockOptions
+		mockEndpoint = nil
+
+		res := meshkitutils.TcpCheck(&endpoint, mockEndpoint)
```

Comment:
Shouldn't we check this after Meshery is deployed?

Reply:
TcpCheck is done by the `meshkit` functions I'm using, `GetServiceEndpoint`.

Reply:
Also, the browser navigation code is the same as the one used for Docker so IDK how it's not working for you here. It definitely works for me for both platforms.

Reply:
I'll change the TcpCheck placement for docker to after the deployment. Thanks!

Reply:
@navendu-pottekkat made the change in my latest commit.

It's failing the `golangci-lint` test despite showing no errors locally. Is there something to fix that?
![image](https://user-images.githubusercontent.com/48997495/117191999-da422b00-adfe-11eb-999b-a2c650d93241.png)